app/javascript/packages/build-sass/cli.js

```diff
@@ -45,6 +45,28 @@ function watchOnce(paths, callback) {
  */
 const isSassException = (error) => 'span' in /** @type {SassException} */ (error);
 
+/**
+ * Returns all file paths contained in the given Sass stack trace.
+ *
+ * @example
+ * ```
+ * 'node_modules/identity-style-guide/dist/assets/scss/uswds/core/_functions.scss 35:8     divide()\n' +
+ * 'node_modules/identity-style-guide/dist/assets/scss/uswds/core/mixins/_icon.scss 77:12  add-color-icon()\n' +
+ * 'app/assets/stylesheets/components/_alert.scss 13:5                                     @import\n' +
+ * 'app/assets/stylesheets/components/all.scss 3:9                                         @import\n' +
+ * 'app/assets/stylesheets/application.css.scss 7:9                                        root stylesheet\n',
+ * ```
+ *
+ * @param {string} sassStack Sass stack trace (see example).
+ *
+ * @return {string[]} Array of file paths.
+ */
+const getErrorSassStackPaths = (sassStack) =>
+  sassStack
+    .split(/\.scss \d+:\d+\s+.+?\n/)
+    .filter(Boolean)
+    .map((basename) => `${basename}.scss`);
```

Comment:
It's not the most elegant thing, but unfortunately Sass doesn't appear to make it available to us in a more structured / machine-readable format. I was inclined to split on the space character and pull the first portion of each line, but I tested and confirmed that files with spaces in them are not marked up in a way to allow us to do reliable space-delimiting.

For example, I renamed `_alert.scss` to `_alert ok.scss`:

```
'node_modules/identity-style-guide/dist/assets/scss/uswds/core/_functions.scss 35:8     divide()\n' +
'node_modules/identity-style-guide/dist/assets/scss/uswds/core/mixins/_icon.scss 77:12  add-color-icon()\n' +
'app/assets/stylesheets/components/_alert ok.scss 13:5                                  @import\n' +
'app/assets/stylesheets/components/all.scss 3:9                                         @import\n' +
'app/assets/stylesheets/application.css.scss 7:9                                        root stylesheet\n',
```